docs/samples/git-setup/clustertemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-v1.yaml

```diff
@@ -0,0 +1,289 @@
+apiVersion: o2ims.provisioning.oran.org/v1alpha1
+kind: ClusterTemplate
+metadata:
+  name: sno-ran-du.v4-Y-Z-1
+  namespace: sno-ran-du-v4-16
+spec:
+  name: sno-ran-du
+  version: v4-Y-Z-1
+  templates:
+    hwTemplate: placeholder-du-template-configmap-v1
+    clusterInstanceDefaults: clusterinstance-defaults-v1
+    policyTemplateDefaults: policytemplate-defaults-v1
+  templateParameterSchema:
+    properties:
+      nodeClusterName:
+        type: string
+      oCloudSiteId:
+        type: string
+      policyTemplateParameters:
+        description: policyTemplateSchema defines the available parameters for cluster configuration
+        properties:
+          sriov-network-vlan-1:
+            type: string
+          sriov-network-vlan-2:
+            type: string
+          sriov-network-pfNames-1:
+            type: string
+          sriov-network-pfNames-2:
+            type: string
+          cpu-isolated:
+            type: string
+          cpu-reserved:
+            type: string
+          hugepages-default:
+            type: string
+          hugepages-size:
+            type: string
+          hugepages-count:
+            type: string
+          install-plan-approval:
+            type: string
+            default: Automatic
```

Comment:
I think we should remove `default: Automatic` from here to avoid confusion, as we define default values in configmaps and the default here is useless.

Reply:
Initially I added it to showcase that the default for the ORAN usecase is `Automatic`. The defaults from the ConfigMaps are defaults for a certain template. But we can remove it, to avoid any possible confusion.